docs/contributing.rst

```diff
@@ -93,7 +93,7 @@ Prerequisites
 
 For running all compiler tests you may want to optionally install a few
 dependencies (`evmone <https://github.com/ethereum/evmone/releases>`_,
-`libz3 <https://github.com/Z3Prover/z3>`_).
+`libz3 <https://github.com/Z3Prover/z3>`_, `Eldarica <https://github.com/uuverifiers/eldarica/>`_).
```

Comment:
Should we list cvc5 as well here? Or is it just not covered by any tests?

Reply:
`cvc5` is currently not used in any SMTChecker tests.
It is used in command-line tests. For those we are gonna need the flag that we discussed.
But I'll add it.